### ISSUE TITLE
Making shapes movable

### ISSUE THREAD

**Issue by saisurya** [websanova/wPaint#35]
Hi,

I'm trying to make the shapes movable once they're drawn on canvas. I understand that you're busy :) Could you give me some guidance on where to start etc.

This is what I'm trying to achieve. 
http://simonsarris.com/blog/510-making-html5-canvas-useful

I need a starting point because I'm a bit confused and lost in wPaint.js code.

Thanks & Regards,
Sai Surya.

**Comment by adunkey** [comment]
I need something similar; that when you load an image it can e moved(and resized) around the canvas? is this possible?

thanks,
ad

**Comment by websanova** [comment]
No, its not possible with this plugin.  Its just a simple paint app.  Doing
moveable shapes is a bit more complex as it involves layering and many
other things.  Sorry, but I don't have any plans for this type of
functionality for the time being.

**Comment by adunkey** [comment]
no problem - thanks for the quick reply. Savage piece of coding you did all the same - thanks!
